tensorflow_hub/compressed_module_resolver_test.py

```diff
@@ -148,6 +148,14 @@ def testAppendFormatQuery(self):
           http_resolver._append_compressed_format_query(handle),
           expected)
 
+  def testGetModulePathTarGz_NoCert(self):
+    os.environ[resolver. _TFHUB_DISABLE_CERT_VALIDATION] = "true"
```

Comment:
Instead of actually setting the environment variable here, let's mock it instead:
```python
with mock.patch.dict(os.environ, {resolver._TFHUB_DISABLE_CERT_VALIDATION: "true"}):
  ...
```